AAI/README.md

```diff
@@ -19,160 +19,98 @@ permalink: aai-introduction
 ### Introduction
 
 The [GA4GH AAI profile specification]({% link AAI/AAIConnectProfile.md %})
-leverages OpenID Connect (OIDC) Servers for use in authenticating the identity of
-researchers desiring to access clinical and genomic resources from [data
+leverages OpenID Connect (OIDC) Servers to authenticate researchers
+desiring to access clinical and genomic resources from [data
 holders]({% link AAI/AAIConnectProfile.md %}#term-data-holder)
-adhering to GA4GH standards, and to enable data holders to obtain security-related
-attributes of those researchers. This is intended to be endorsed as a GA4GH standard,
-implemented by GA4GH Driver Projects, and shared broadly.
-
-To help assure the authenticity of identities used to access data from GA4GH
-Driver Projects, and other projects that adopt GA4GH standards, the Data Use and
-Researcher Identity (DURI) Work Stream has developed a standard around
-[claims](https://github.com/ga4gh-duri/ga4gh-duri.github.io/tree/master/researcher_ids).
-This standard assumes that some GA4GH Claims provided
-by Brokers described in this document will conform to the DURI researcher-identity
-policy and standard. This standard does NOT assume that DURI's GA4GH Claims will be
-the only ones used.
-
-This AAI standard aims at developing an approach that enables data holders’ and data owners 
-systems to have systems that recognize and accept identities from multiple Brokers -- allowing for
-a federated approach. An organization can still use this specification and not
-support multiple Brokers, though they may find in that case that it’s just using
-a prescriptive version of OIDC.
+adhering to GA4GH standards. Beyond standard OIDC authentication, AAI enables
+data holders to obtain security-related attributes and authorizations of those
+researchers.
+
+The Data Use and Researcher Identity (DURI) Work Stream has developed standard
+[claims](https://github.com/ga4gh-duri/ga4gh-duri.github.io/tree/master/researcher_ids)
+for representing common researcher authorizations and attributes. This standard
+assumes that GA4GH Claims provided by Brokers described in this document
+MAY conform to the DURI researcher-identity policy and standard. This standard
+does NOT assume that DURI's GA4GH Claims will be the only ones used.
+
+#### Technical Summary
+
+At its core, the AAI specification defines cryptographically secure tokens for exchanging
+researcher attributes called [Visas](aai-openid-connect-profile#term-visa), and how various
```

Comment:
"signed researcher attributes"?  "individually signed researcher attributes"?.. (I guess at this point I might be thinking - how is this different from regular oauth claims/attributes?).. and the signed visas is to me the distinguishing feature of the spec (when it comes to claims)
(this is just me thinking out loud - feel free to ignore!)